lib/core/datatable/components/datatable/datatable.component.html

```diff
@@ -77,6 +77,7 @@
                 <div *ngIf="multiselect" class="adf-datatable-table-cell adf-datatable-table-checkbox">
                     <mat-checkbox
                         [checked]="row.isSelected"
+                        [attr.aria-label]="idx+1"
```

Comment:
I don't think it's correct. The screen-reader will read it as "1" for example, without any clue what exactly the user tabbed to.

Reply:
Any suggestions for what should put in that aria-label? Maybe "row x selected"?